issues with ffprobe showFormat and multiline tags

I came across a file which seems to have tags that contain values that span multiple lines. This seems to cause the DataParser to error:

```
Caused by: java.lang.RuntimeException: key=value was expected but got: 
	at com.github.kokorin.jaffree.ffprobe.data.DataParser.parseLine(DataParser.java:54)
	at com.github.kokorin.jaffree.ffprobe.data.DataParser.parse(DataParser.java:151)
	at com.github.kokorin.jaffree.ffprobe.FFprobeResultReader.read(FFprobeResultReader.java:40)
	at com.github.kokorin.jaffree.ffprobe.FFprobeResultReader.read(FFprobeResultReader.java:32)
	at com.github.kokorin.jaffree.process.ProcessHandler$3.run(ProcessHandler.java:186)
	at com.github.kokorin.jaffree.process.Executor$1.run(Executor.java:49)
```
with showFormat enabled:
```
ffmpeg.setShowFormat(true)
```
running ffprobe with options `ffprobe -loglevel 16 -show_format -show_private_data` shows this output (long text shortened with `...`):
```
[FORMAT]
filename=/tmp/cvma7ou0kk5infmok4403378daciqkia/cvma7ou0kk5infmok4403378daciqkia.mp3
nb_streams=2
nb_programs=0
format_name=mp3
format_long_name=MP2/3 (MPEG audio layer 2/3)
start_time=0.011995
duration=122.044082
size=2029181
bit_rate=133012
probe_score=51
TAG:comment=The most popular...way.

Samples...provide.

Then,...format
TAG:encoder=Lavf56.40.101
[/FORMAT]
```

I was thinking about modifying the dataParser so that it if it does not find a new tag, it will add the content to the previous value. However, that probably will blow up as easy as the multiline value containing a `=`. Maybe a better approach would be to only parse values we know are relevant for ffprobes result and ignore everything else? Or we just ignore multiline tags, by simply returning instead of throwing an error? what do you think?

The file is available here https://www.dropbox.com/s/wsczg7umwfx0isc/multiline.mp3?dl=0

I was also having some issues running the tests locally, mostly because I was unable to find the input files required for testing. Are they available for download somewhere?

--- a/src/main/java/com/github/kokorin/jaffree/ffprobe/data/DataParser.java
+++ b/src/main/java/com/github/kokorin/jaffree/ffprobe/data/DataParser.java
@@ -17,9 +17,15 @@
 
 package com.github.kokorin.jaffree.ffprobe.data;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import java.util.*;
 
 public class DataParser {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(DataParser.class);
+
     // State
     private final Deque<State> stack;
 
@@ -51,7 +57,8 @@ public class DataParser {
             } else {
                 String[] keyValue = line.split("=", 2);
                 if (keyValue.length != 2) {
-                    throw new RuntimeException("key=value was expected but got: " + line);
+                    LOGGER.warn("failed to parse line, which will be ignored: {}", line);
+                    return;
                 }
                 String key = keyValue[0];
                 String value = keyValue[1];

